src/app/shared/file-upload/file-upload.component.ts

```diff
@@ -9,15 +9,23 @@ import { Component, Input, Output, EventEmitter } from '@angular/core';
   templateUrl: './file-upload.component.html',
   styleUrls: ['./file-upload.component.scss']
 })
-export class FileUploadComponent {
+export class FileUploadComponent implements OnInit {
 
   /** Form field flex dimension */
   @Input() flex: any;
+  @Input() acceptFilter: string;
+
   /** Selected file name */
   fileName: File;
 
   constructor() { }
 
+  ngOnInit() {
+    if (!this.acceptFilter) {
+      this.acceptFilter = '.xls,.xlsx,.pdf,.doc,.docx,.txt,.png,.jpeg,.jpg';
```

Comment:
I dont think "txt" is by default enabled on the backend

Reply:
Done! removed the txt file extension